### PR TITLE
tests: add test for pgsql probe bug 6080 - v1

### PR DESCRIPTION
**Pull request by jufajardini**
Add test for pgsql probing function bug 6080.
Crafted pcap.

Related to
Bug #6080

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/6080

### CODE REVIEW

**Comment by jufajardini** [comment]
Force-pushed to include the scapy script, which wasn't originally the folder.

File: tests/pgsql-bug-6080-probe-test-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      dest_port: 5432
+      event_type: pgsql
+      pgsql.tx_id: 1
+      proto: TCP
+      src_ip: 172.16.1.1
+      src_port: 1050
+- filter:
+    count: 0
+    match:
+      app_proto: pgsql
```
Comment (catenacyber) [inline tests/pgsql-bug-6080-probe-test-01/test.yaml:18]: We do _not_ want the flow to be recognized as pgsql, do we ?

Comment (catenacyber) [reply tests/pgsql-bug-6080-probe-test-01/test.yaml:18]: So, it should be there is 1 event_type flow, and 0 with `app_proto: pgsql` and no other fields to test

File: tests/pgsql-bug-6080-probe-test-01/test.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+args:
+- -k none
```
Comment (jufajardini) [inline tests/pgsql-bug-6080-probe-test-01/test.yaml:2]: Add min-version check.

**Comment by jufajardini** [comment]
Replaced by: https://github.com/OISF/suricata-verify/pull/1502

**Pull request closed without merging**